databricks_cli/configure/cli.py

```diff
@@ -44,6 +52,27 @@ def _configure_cli_token(profile, insecure):
     update_and_persist_config(profile, new_config)
 
 
+def _configure_cli_aad_token(profile, insecure):
+    config = ProfileConfigProvider(profile).get_config() or DatabricksConfig.empty()
+    host = click.prompt(PROMPT_HOST, default=config.host, type=_DbfsHost())
+
+    is_token_env_set = click.prompt(PROMPT_ENV_TOKEN, type=bool)
```

Comment:
Is prompting in this function twice helpful? Can't we just check the `ENV_TOKEN` and `ENV_AZ_TOKEN` keys in environ without the prompt and save the user a couple of keystrokes?

Reply:
Removed this and changed workflow